user/views.py

```diff
@@ -14,3 +25,115 @@ class UserRetrieveDeleteView(generics.RetrieveDestroyAPIView):
     lookup_field = 'pk'
     queryset = User.get_all_users()
     serializer_class = UserSerializer
+
+
+class SendOTPView(views.APIView):
+    permission_classes = (AllowAny,)
+
+    def post(self, request: Request):
+        email = request.data.get('email')
+        client_id = request.data.get('client_id')
+
+        client = Application.objects.get(client_id=client_id)
```

Comment:
Will raise an exception. Please use `filter(...).first()` and proceed ahead only if `client` is not `None`.